packages/desktop-ui-lib/src/lib/auth/services/auth.service.ts

```
@@ -60,4 +52,118 @@
 			params: { permission }
 		});
 	}
+
+	confirmEmail(body: IUserEmailInput & IUserTokenInput): Observable<Object> {
```

Biome

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)